packages/node_modules/@ciscospark/widget-recents/src/components/spaces-list/index.js

```diff
@@ -0,0 +1,24 @@
+import React, {PropTypes} from 'react';
```

Comment:
Yea, I'm keeping them out because as they are, they're not really generic enough to be standalone, and completely useless without the main widget-recents container.